Allow optional to be false

padraicc opened the pull request:
Allow 'optional: false' for the algorithm commands. If optional is set to false, it does not write the optional commands that come after, as they are not needed.

padraicc committed:
Allow optional to be false

Allow 'optional: false' for the algorithm commands. If set to optional is set to false it does not write the optional commands that come after, as they are not needed.

henryjuho commented:
Would it be possible to add keyword arguments for the individual optional parameters?

padraicc committed:
Added keyword args for optional alg command

Added the keyword args that are set to the default values as specified in the anavar 1.2.1 manual.

henryjuho commented:
Should false be default?

padraicc commented:
Yes.

padraicc committed:
Default on set_alg_opts changed to False
